mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -403,6 +423,115 @@ impl NexusChild {
         })
     }
 
+    pub async fn probe_index(
```

Comment:
Please document what the function does, and make it a good habit, please. And should this be pub?